services/download/src/test/resources/db/testdata-enforce-download-again/V99__createTestDataForDownloadTodayAgainIntegrationTest.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO federation_batch_info(batch_tag, date, status) VALUES ('batch1_tag', current_timestamp, 'PROCESSED');
```

Comment:
Even though this approach of using flyway for initial test data is fine with me, there is an alternative to it. We can also use @Sql("/your_script.sql") annotation to load the sql statements needed for this test.  As you prefer.